test/scale-infra/deploy_ks_core.yaml

```diff
@@ -0,0 +1,104 @@
+---
+- hosts: masters
+  remote_user: ubuntu
+  become: yes
+  gather_facts: no
+  vars:
+    ks_release:  "{{ ks_release }}"
+  tasks:
+    - name: install jq and make
+      shell: |
+             sudo apt update
+             sudo apt install jq make -y
+
+    - name: install golang
+      shell: |
+             curl -LO https://go.dev/dl/go1.24.0.linux-amd64.tar.gz
+             rm -rf /usr/local/go && tar -C /usr/local -xzf go1.24.0.linux-amd64.tar.gz
+             export PATH=$PATH:/usr/local/go/bin
+             go version
+                  
+    - name: install yq
+      shell: |
+             sudo wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64
+             sudo chmod a+x /usr/local/bin/yq
+
+    - name: install OCM CLI
+      shell: curl -L https://raw.githubusercontent.com/open-cluster-management-io/clusteradm/main/install.sh | bash
+        
+    - name: install KubeFlex
```

Comment:
A little more specifically, this is installing the KubeFlex _CLI_, not installing the KubeFlex mechanism in the KubeFlex hosting cluster.

Reply:
Fixed! Updated the task with the following name instead: `installing the KubeFlex CLI`